flask_scraper_demo/app/scrapers/execute_search.py

```diff
@@ -3,27 +3,30 @@
 from .ifc_scraper import scrape_ifc
 from .worldbank_scraper import scrape_world_bank
 
-SCRAPERS = [
-    (scrape_ifc, 'IFC'),
-    (scrape_world_bank, 'World Bank'),
+
+SELECT_ALL_NAME = 'All'
+
+SCRAPER_MAP = {
+    'IFC': scrape_ifc,
+    'World Bank': scrape_world_bank,
 
     # NOTE: Uncomment as these are implemented.
-    # (scrape_miga, 'MIGA'),
-    # (scrape_afdb, 'AfDB'),
-    # (scrape_eib, 'EIB'),
-    # (scrape_ebrd, 'EBRD'),
-    # (scrape_adb, 'ADB'),
-    # (scrape_aiib, 'AIIB'),
-    # (scrape_idb, 'IDB'),
-    # (scrape_opic, 'OPIC'),
-    # (scrape_fmo, 'FMO'),
-    # (scrape_cdc, 'CDC'),
-    # (scrape_bio, 'BIO'),
-    # (scrape_kfw, 'KfW')
-]
-
-
-def execute_search(search_term):
+    # 'MIGA': scrape_miga,
+    # 'AfDB': scrape_afdb,
+    # 'EIB': scrape_eib,
+    # 'EBRD': scrape_ebrd,
+    # 'ADB': scrape_adb,
+    # 'AIIB': scrape_aiib,
+    # 'IDB': scrape_idb,
+    # 'OPIC': scrape_opic,
+    # 'FMO': scrape_fmo,
+    # 'CDC': scrape_cdc,
+    # 'BIO': scrape_bio,
+    # 'KfW': scrape_kfw,
+}
```

Comment:
I changed the data structure to a `dict` so that this automatically enforces that all names are unique